Flaky: TestRelayWithInvalidMessagesSteady fails on transient EL dial timeout

**Claude:** This issue was filed by Claude (AI assistant) based on CircleCI flake data and failure artifacts.

## Test

- **Name:** `TestRelayWithInvalidMessagesSteady`
- **Package:** `github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop/loadtest`
- **Test file:** `op-acceptance-tests/tests/interop/loadtest/invalid_msg_test.go`
- **Assertion site:** `op-acceptance-tests/tests/interop/loadtest/schedule.go:273`

## Recent flakes (CircleCI Insights)

| Pipeline | Job | Workflow date |
|----------|-----|---------------|
| [123327](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/123327/workflows/c6e21643-0e34-4615-ae7c-fc05f7ee0ea9/jobs/4891215) | `memory-all-kona-op-reth` | 2026-04-23 |
| [123217](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/123217) | `memory-all-opn-op-reth` | 2026-04-22 |
| [123082](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/123082) | `memory-all-kona-op-reth` | 2026-04-21 |

## Failure

```
assertions.go:387:
    Error Trace: op-acceptance-tests/tests/interop/loadtest/schedule.go:273
    Error:       Received unexpected error:
                 dial tcp 127.0.0.1:45777: i/o timeout
--- FAIL: TestRelayWithInvalidMessagesSteady (181.92s)
```

Port `45777` is a proxy in front of `l2B`'s `op-reth` EL.

## Likely root cause

`Steady.Run` in `op-acceptance-tests/tests/interop/loadtest/schedule.go:251-296` spawns a polling goroutine that calls `s.el.InfoByLabel(t.Ctx(), eth.Unsafe)` every `blockTime` to sample gas usage for AIMD backpressure. On any non-context error it does `t.Require().NoError(err)` (line 273), which fails the whole test.

`TestRelayWithInvalidMessagesSteady` simultaneously spams 1,000 invalid executing messages per block time at the same `l2B` EL via the same proxy. Under that load, a transient `dial tcp 127.0.0.1:45777: i/o timeout` (e.g. TCP accept backlog saturation on the proxy) can occur. The check `errors.Is(err, t.Ctx().Err())` on line 270 only catches context cancellation, not transient network errors while the test is still running.

## Why flaky (not consistent)

The dial timeout is load/timing dependent. The poll fires roughly once per 2s, and the proxy/EL only saturates intermittently enough for a dial to time out. Every other goroutine in this test treats EL errors as non-fatal — only this Steady poll treats them as fatal.

## Proposed fix

Make the Steady polling goroutine resilient to transient EL errors. Its purpose is to sample gas usage for backpressure, not to assert EL liveness. Log a warning, skip the adjustment, continue polling — matching the tolerance pattern in `schedule.go:325-333` and `invalid_msg_test.go:77`.

--- a/op-acceptance-tests/tests/interop/loadtest/schedule.go
+++ b/op-acceptance-tests/tests/interop/loadtest/schedule.go
@@ -270,7 +270,11 @@ func (s *Steady) Run(t devtest.T, spammer Spammer) {
 					if errors.Is(err, t.Ctx().Err()) {
 						return
 					}
-					t.Require().NoError(err)
+					// A transient sampling failure (e.g. dial timeout under load) is
+					// not a test failure. Log and try again next tick; leave the AIMD
+					// state unchanged so the current RPS is preserved.
+					t.Logger().Warn("Steady sampler failed to read unsafe head", "err", err)
+					continue
 				}
 				gasTarget := unsafe.GasLimit() / s.elasticityMultiplier
 				// Apply backpressure when we meet or exceed the gas target.

